queue/redis/opts.go

```diff
@@ -40,7 +40,7 @@ func WithChannels(channels ...string) ClientOpt {
 		}
 
 		// set the queue channels in the redis client
-		c.config.Channels = channels
+		c.SetRoutes(channels)
```

Comment:
do you think we should take the `routes` terminology all the way and update queue to use `routes` instead of `channels`. seems a little odd to see both here.

Reply:
yeah i agree its confusing. they really should be called `keys` in the redis client _only_, not routes or channels. so we may as well go with all routes

Reply:
done! `channels` are now `routes`